lib/services/chatwork.rb

```diff
@@ -0,0 +1,56 @@
+require 'json'
+
+class Service::ChatWork < Service::Base
+  title 'ChatWork'
+
+  string :api_token, :placeholder => 'API Token',
+         :label => 'Your ChatWork API Token. <br />' \
+                   'You can create an API Token ' \
+                   '<a href="https://www.chatwork.com/service/packages/chatwork/subpackages/api/apply_beta_business.php">here</a>.'
+  string :room, :placeholder => 'Room ID',
+         :label => 'Specify the Room ID where you want to send messages.<br />' \
+                   'Room ID is the numeric id shown in the URL of each group chat.<br />' \
+                   'For example: if the URL of the group chat where you want to send messages ' \
+                   'is chatwork.com/#!rid00000, copy "00000", and paste it here.'
+
+  page 'API Token', [:api_token]
+  page 'Room ID', [:room]
+
+  def receive_verification(config, _)
+    send_message(config, receive_verification_message)
+    [true, "Successfully sent a message to room #{config[:room]}"]
+  rescue => e
+    log "Rescued a verification error in ChatWork: #{e}"
+    [false, "Could not send a message to room #{config[:room]}. #{e.message}"]
+  end
+
+  def receive_issue_impact_change(config, payload)
+    send_message(config, format_issue_impact_change_message(payload))
+  end
+
+  private
+
+  def receive_verification_message
+    'Boom! Crashlytics issue change notifications have been added. For more info: ' \
+    'http://support.crashlytics.com/knowledgebase/articles/349341-what-kind-of-third-party-integrations-does-crashly'
```

Comment:
I removed the anchor tag markup around this link as this was being escaped in the ChatWork message.